src/WebJobs.Script/Content/Script/functions.js

```diff
@@ -69,6 +70,11 @@ function createFunction(f) {
         inputs.unshift(context);
         delete context._inputs;
 
+        if (context._http) {
+            context.res = res();
```

Comment:
Here's the question:  do we want to make this a breaking change and force everyone to migrate to using the express-like api & response objects?

Otherwise we'll still need to assume that a returned object could be poco or a response object via the above logic, I don't see a way around it.